setup.py

```diff
@@ -1,8 +1,7 @@
-# -*- encoding: utf8 -*-
 from setuptools import setup, find_packages
 from setuptools.command.test import test
 
-from sorl import __version__, __author__, __maintainer__, __email__, __license__
+from sorl import __author__, __maintainer__, __email__, __license__
```

Comment:
Let's remove this import completely and move all metadata inside this file (as preparation for stop using setup.py at some point completely).

Reply:
@jezdez do you mean let's move all the metadata

1) from __init__.py to setup.py or
2) from setup.py to __init__.py?

I would like to migrate from using setup.py to pyproject.toml as per PEP518 in Jazzband projects at some point in time, but that will be larger migration and not in the scope of this PR :)

Reply:
The first, for the same reason as you mention. setuptools_scm [nowadays supports pyproject.toml files](https://github.com/pypa/setuptools_scm/#pyprojecttoml-usage), but it's a bit too early to adopt it just yet. Let's stay with setup.py for a bit more and let the early bugs with it be figured out.

Reply:
Sure, the first was what I was thinking too, just wanted to make sure.